src/lib.rs

```diff
@@ -718,6 +718,9 @@ impl Installer {
         let configure_dir = TempDir::new_in(mount_dir.join("tmp"), "distinst")?;
         let configure = configure_dir.path().join("configure.sh");
 
+        // Pop!_OS does not need the retain workaround.
+        let retain = if iso_os_release.name == "Pop!_OS" { &[] } else { retain };
```

Comment:
Is there any reason not to adjust the function generating the retain list for Pop!_OS? I don't like this kind of code as it introduces two different pathways

Reply:
It's a bit difficult since the squashfs image hasn't been extracted yet, at that time.

Reply:
Since the retain list is dependant on the image being installed, I think we must have access to the extracted image before we can make any decisions

Reply:
I'll move the logic around so that it gets the manifest after.

Reply:
Any reason Installer::initialize needs to have the list?

Reply:
check-language-support also needs to run inside the chroot

Reply:
`initialize()` had the list so that the packages listed in the remove manifest would not be added as a package to be removed.

Reply:
I think the list should be modified in configure then